Automattic/sensei
# Update npm dependencies

Resolves #7692 #6123

## Proposed Changes

Sorry for the huge PR, but all the things were depending on each other, so it's complicated to separate in small ones. Notice that this big commit is related to lint fixes: https://github.com/Automattic/sensei/pull/7695/commits/82ca947db8de772a4123db6d2ba5951d18934256t

* Update dependencies.
* Clean up some dependencies.
* Fix lint.
* Fix unit tests.
* Fix dev configurations.
* Fix styles.

## Testing Instructions


1. Build the project and make sure it works.
2. Navigate through the Sensei LMS features (especially Import/Export), and make sure the styles continue working properly.
3. Check that tests and lint are passing.

#### Students menu (#6123)

1. Go to Sensei LMS » Students
2. You must have a couple of students. Click on the 3 dots at the end of any student.
3. Make sure the menu opens normally and the styles are not broken.

#### Sensei Pro

- See https://github.com/Automattic/sensei-pro/pull/2644, and make sure Sensei Pro builds properly.

## Pre-Merge Checklist
